back/app/controllers/web_api/v1/idea_statuses_controller.rb

```diff
@@ -19,4 +68,36 @@ def set_idea_status
     @idea_status = IdeaStatus.find(params[:id])
     authorize @idea_status
   end
+
+  def serialize(record_or_records)
+    WebApi::V1::IdeaStatusSerializer.new(record_or_records).serializable_hash
+  end
+
+  def apply_index_filters
+    return if params[:participation_method].blank?
+
+    @idea_statuses = @idea_statuses.where(participation_method: params[:participation_method])
+  end
+
+  def idea_status_params_for_create
+    params_attrs = [:participation_method, *shared_params]
+    params_attrs << :code if IdeaStatus::AUTOMATIC_STATUS_CODES.exclude?(params.dig(:idea_status, :code))
+    params.require(:idea_status).permit(params_attrs)
+  end
+
+  def idea_status_params_for_update
+    params_attrs = shared_params
+    if !@idea_status.automatic? && IdeaStatus::AUTOMATIC_STATUS_CODES.exclude?(params.dig(:idea_status, :code))
```

Comment:
Same remark here, feels like an "abuse" of the strong params, where we're mixing business logic with requests sanitization logic.